smile_id_core/Utilities.py

```diff
@@ -88,6 +85,8 @@ def __configure_job_query(self, user_id, job_id, options, sec_params):
             "user_id": user_id,
             "image_links": options.get("return_images"),
             "history": options.get("return_history"),
+            "source_sdk": "PYTHON",
+            "source_sdk_version": "2.0.0",
```

Comment:
I see we have version defined in multiple place, it wil be nice to extract this into a constant